modules/ld/src/main/scala/ch/epfl/bluebrain/nexus/admin/ld/IdRefBuilder.scala

```diff
@@ -0,0 +1,36 @@
+package ch.epfl.bluebrain.nexus.admin.ld
+
+import ch.epfl.bluebrain.nexus.admin.ld.Prefix.randomPrefixName
+import ch.epfl.bluebrain.nexus.admin.refined.ld._
+
+/**
+  * Helper to build a [[IdRef]]
+  *
+  * @param name  the prefix name
+  * @param value the value to which the name expands
+  */
+class IdRefBuilder(name: PrefixName, value: PrefixValue) {
```

Comment:
any reason this is a separate type from Prefix?

Reply:
For readability when building it on ``Const.scala``. Otherwise it becomes very verbose.
In any case, that't why there is a `apply` method where you can use t from a prefix

Reply:
Fair point.